docs/en/common_usage/mode_analysis.md

```diff
@@ -0,0 +1,740 @@
+# Count the calculation of Model and Parameter
+
+1. Define Model
+
+```python
+import torch.nn.functional as F
+import torchvision
+from mmengine.model import BaseModel
+
+class MMResNet50(BaseModel):
+    def __init__(self):
+        super().__init__()
+        self.resnet = torchvision.models.resnet50()
+
+    def forward(self, imgs, labels=None, mode='tensor'):
+        x = self.resnet(imgs)
+        if mode == 'loss':
+            return {'loss': F.cross_entropy(x, labels)}
+        elif mode == 'predict':
+            return x, labels
+        elif mode == 'tensor':
+            return x
+```
```

Comment:
Need 2 blank indentations

Reply:
mdformat.................................................................Failed
- hook id: mdformat
- files were modified by this hook

if using 2 blank indentation, pre-commit errors. i find that some docs using 3 blank indentation. so i used

Reply:
OK